boards/nucleo-f767zi/Makefile.include

```diff
@@ -1,2 +1,11 @@
 # load the common Makefile.include for Nucleo boards
 include $(RIOTBOARD)/common/nucleo144/Makefile.include
+
+# openocd configuration file for `stm32f7` relies on probing to find out
+# FLASH_ADDR. On this board probing (`flash probe 0`) fails when `srst` is
+# asserted, but `srst` needs to be asserted to be able to flash the `BOARD`
+# when sleeping or after a hardfault.
+# To circumvent this in boards/common/stm32/dist/stm32f7.cfg we define a new
+# flash bank with the appropriate flash start address and specify that this is# the flash bank to be used as default configuration instead of the default (1)
```

Comment:
```suggestion
# flash bank with the appropriate flash start address and specify that this is
# the flash bank to be used as default configuration instead of the default (1)
```